source/NationalInstruments/jquery.flot.intensitygraph.js

```diff
@@ -177,6 +178,26 @@ THE SOFTWARE.
                 return palette;
             };
 
+            function findNearbyItems (plot, canvasX, canvasY, series, seriesIndex, radius, computeDistance, items) {
```

Comment:
Are we comfortable with this implementation outright ignoring the radius and computeDistance parameters?

Reply:
Yeah, that was a concern of mine as well. But if we want the algorithm to act the way it does currently (find what value the mouse is currently over, instead of what's nearby), then `radius` and `computeDistance` don't really make sense. On the other hand, it's possible that someone could want `findNearbyItems` to return multiple intensity graph values that are within a radius of the mouse position. What's your opinion on how exactly this should act?

Reply:
For what it's worth, I worked out an alternate solution that can return multiple values and uses `radius` and `computeDistance`. Let me know which implementation you prefer.
![image](https://user-images.githubusercontent.com/5454342/89434395-1bd76580-d709-11ea-9b19-a93784af6dd5.png)

Reply:
I think this makes sense. I don't know why we wouldn't want to support this.